HHS/Head-Start-TTADP
# [TTAHUB-3402] Hook up QA widgets to BE

## Description of change

This hooks up the new QA dashboard to the SSDI.

## How to test

- Review the code
- Make sure all widgets are displaying correctly
- Verify filters work correctly
- Sorting and exporting should work as it does on other pages


## Issue(s)

* https://jira.acf.gov/browse/TTAHUB-3402
* https://jira.acf.gov/browse/TTAHUB-3457
* https://jira.acf.gov/browse/TTAHUB-3458
* https://jira.acf.gov/browse/TTAHUB-3459


## Checklists

### Every PR



### Before merge to main


### Production Deploy


### After merge/deploy
